### DISCUSSION TITLE
Integration with smaily.ee

### DISCUSSION THREAD

**Discussion by leyakoreya** [hizzle-co/noptin#600] [Ideas]
Integration with [smaily.ee](https://smaily.com/)
I use services to send from [smaily.ee](https://smaily.com/)

and is not yet ready to completely abandon their services, although you offer similar ones, but do not have such visual solutions for creating a mailing list and some functions such as Happy Birthday, abandoned cart. Therefore, it is possible to start with a better integration to use.

**Comment by picocodes** [comment]
This has been built:- https://noptin.com/integrations/connect-smaily-to-wordpress/
